cmd/core/cmd.go

```diff
@@ -61,6 +61,10 @@ func generateCommandFunc(cmdEntry CommandEntry) func(cmd *cobra.Command, args []
 		}
 
 		adminCfg := admin.GetConfig(ctx)
+		if len(adminCfg.Endpoint.String()) == 0 {
+			return cmdEntry.CmdFunc(ctx, args, CommandContext{})
```

Comment:
I do not think this is the solution right. What we want is that client should be lazy to connect and ignore if needed

Reply:
Do you mean that some of the commands (like sandbox, upgrade) should not connect to FlyteAdmin no matter if `adminCfg.Endpoint` is set or not?

Reply:
@kumare3 we can have current solution implemented and then take up the lazy initialization as an enhancement. @pingsutw you can create a separate issue for lazy initialization for the client.
This can be interim solution